### PR TITLE
Create emergency-shutdown.md

### PR DESCRIPTION
**Pull request by Davidutro**

### CODE REVIEW

File: faqs/emergency-shutdown.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+## Why is there a time period before CDP owners can redeem collateral?
+
+Calculating CDP claims involves on-chain calculation and movement of funds. Processing may take a long time depending on the congestion on the network, therefore the default time period is set to 6 hours. This delay period is also a precaution for unknown circumstances that may arise.
+
+## Can MKR voters set the collateral claim processing time period to 0?
+
+Setting a shorter time period will expose CDP owners to the risk of incorrect collateral claim amounts. CDP owners who try to close their CDPs and exit from PETH to WETH before the processing is finished will likely penalize themselves, the delay period is intended to protect them.
+
+When Emergency Shutdown happens W-ETH is taken from the `tub`, which is all the WETH that is pooled into PETH, to cover Dai holders. This lowers the PETH ratio. If a user tries to convert their PETH into WETH during the Emergency Shutdown's 6 hour delay period they may be subject to an artificially low PETH ratio.
```
Comment (ghettodev) [inline faqs/emergency-shutdown.md]: Please change W-ETH to WETH and remove the reference to `tub`.

**Comment by ghettodev** [comment]
Please update the SUMMARY.md file as well.

**Comment by Davidutro** [comment]
@ghettodev changes made
updating Summary.md after the risk management faq is accepted as well

**Comment by ghettodev** [comment]
Why are you updating the summary after?

File: faqs/emergency-shutdown.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+The PETH ratio drops instantaneously when an Emergency Shutdown is triggered and comes back up as CDPs are processed. PETH holders should wait the 6 hour delay period like CDP owners.  
+
+## If the prices keep falling after the price feeds are frozen, doesn't this mean that people redeeming Dai and CDPs may get less than $1 of collateral?
+
+Yes, it is possible for users to get less than a dollar worth of collateral. Dai holders are able to immediately redeem their Dai for collateral and may experience some slippage. CDP owners must wait the 6 hour delay period for claims to be calculated and are therefore exposed to the risk of the underlying collateral price moving during these 6 hours.
+
+## How quickly can an Emergency Shutdown be performed?
+
+To trigger an Emergency Shutdown, an Executive Vote needs to be voted on by MKR voters. The proposal needs to garner enough MKR until it beats the previous proposal, in line with [continuous approval voting](https://github.com/makerdao/community/blob/master/faqs/governance.md#what-is-continuous-approval-voting). Therefore, it depends on how quickly MKR voters can act to beat the previous winning proposal.
```
Comment (ghettodev) [inline faqs/emergency-shutdown.md]: Please use relative links.

File: faqs/emergency-shutdown.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+-   Changes in the distribution of the MKR token
+
+-   Changes in the Risk Parameters
+
+## Who decides how the system should be redeployed?
+
+Since it is open source software, anyone can decide. Ideally, the parameters of redeployment should depend on the reason for the Emergency Shutdown, and should not be altered unilaterally and arbitrarily. Here is a rough example of a framework for making changes on redeployment:
+
+| Reason            | Solution                                                                                                                                                                        |
```
Comment (ghettodev) [inline faqs/emergency-shutdown.md]: You don't need to space tables manually. 

```
| Reason | Solution |
|---|---|
| Governance attack | Fork out malicious MKR holders in new redistribution,  redeploy system with everything else as-is|
```

File: faqs/emergency-shutdown.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+Yes, an Emergency Shutdown has been [performed](https://www.reddit.com/r/MakerDAO/comments/7lhbmx/psa_sai_now_globally_settled/) on the Sai system, before Single Collateral Dai.
+
+## How was testing for Emergency Shutdown done?
+
+Emergency Shutdown was originally introduced and tested with the Sai prototype deployments. Besides the tests in [the code](https://github.com/makerdao/sai/blob/master/src/sai.t.sol) itself, there were three live runs, all of which happened on deployments of the Sai prototype on the Ethereum Mainnet.
+
+## How will CDP users and Dai holders be notified of Emergency Shutdown?
+
+There will be notices within all official Maker front-ends, as well as announcements on our blog and on all our official communication channels like Twitter, Reddit, Medium, Rocketchat, Telegram, WeChat, and Kakaotalk.
```
Comment (ghettodev) [inline faqs/emergency-shutdown.md]: Please add a single hard return at the end of the file

File: faqs/emergency-shutdown.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+## What is an Emergency Shutdown?
+
+One important security feature of Dai is called Emergency Shutdown, formerly known as Global Settlement. This crucial security feature allows the system to shut down and make underlying collateral available for redemption by [Dai](https://github.com/makerdao/community/blob/master/faqs/dai.md) holders and CDP owners.
+
+## What happens during an Emergency Shutdown?
+
+1. Emergency Shutdown is activated:
+If MKR voters believe that the system is subject to a serious attack, or if an Emergency Shutdown is scheduled as part of a technical upgrade, they can activate an Emergency Shutdown. This stops CDP creation and freezes the Price Feed.
+
+2. Collateral Claims are processed:
```
Comment (ghettodev) [inline faqs/emergency-shutdown.md]: ordered list items should be repeating numbers: https://github.com/makerdao/community/blob/master/contributing/style-guide.md

File: faqs/emergency-shutdown.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
+## Who can redeploy the system?
+
+Anyone can redeploy the system since it is open source software.
+
+## What if multiple redeployments happen, who decides which one is legitimate?
+
+Depending on the details of each redeployment, the market, along with various system participants will have to choose the one with the most appropriate changes. Some factors include:
+
+-   No unwarranted code changes
```
Comment (ghettodev) [inline faqs/emergency-shutdown.md]: unordered list items should be asterisk and single space. https://github.com/makerdao/community/blob/master/contributing/style-guide.md#lists

File: faqs/emergency-shutdown.md [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+| Oracle Attack     | Fork out Oracle module for a new one with a vulnerability fix,  redeploy system with everything else as-is                                                                      |
+| Market Black Swan | Redeploy system with everything as-is, allow MKR voters to decide  how best to address this event through new or improved system mechanics  that can be added post-redeployment |
+
+## During Emergency Shutdown, will CDP owner's collateral be given away to other people?
+
+During Emergency Shutdown, collateral is immediately made available for redemption by Dai holders. This means that someone may receive a part of your original collateral. However, after the time period where CDP collateral claims are processed, any CDP owner will be able to withdraw collateral that represents the full net value of their CDP.
+
+Example:
+
+If you have a CDP with 1 ETH that's worth 300 USD, and 100 debt, you will be left with 0.6666 ETH and no debt. The Net Value of the CDP is made available for redemption, which is 300 USD value of the collateral, minus 100 USD of debt. Therefore, you will be able to claim 0.6666 ETH after the 6 hour delay period.
```
Comment (ghettodev) [inline faqs/emergency-shutdown.md]: Examples should be code fenced:

https://github.com/makerdao/community/blob/master/contributing/style-guide.md#faq-style-guide

File: faqs/emergency-shutdown.md (hunk 3 above)
Comment (Davidutro) [reply faqs/emergency-shutdown.md]: autogenerated the table

File: faqs/emergency-shutdown.md (hunk 4 above)
Comment (Davidutro) [reply faqs/emergency-shutdown.md]: done

File: faqs/emergency-shutdown.md (hunk 5 above)
Comment (Davidutro) [reply faqs/emergency-shutdown.md]: done

File: faqs/emergency-shutdown.md (hunk 6 above)
Comment (Davidutro) [reply faqs/emergency-shutdown.md]: fixed

**Comment by Davidutro** [comment]
@ghettodev fixed all changes. 
unsure if the code fencing is meant to be that way.

**Comment by ghettodev** [comment]
If there is a long line, no coloured syntax or you don't need to preserve unusual formatting then a `>` works.

**Pull request merged**